### ISSUE TITLE
getting same accuracy on every epoch, please let me know if there is any solution

### ISSUE THREAD

**Issue by avinash2222** [zhixuhao/unet#133]
i have 4k images, no of epoch=10, adam(.0001) learning rate, binary crass entropy, unet model

**Comment by tuanphan09** [comment]
First, I think you should downsampling or crop your image base on your problem.

Second, I used this repo's model to train with my own data and I had some problems on the result. It's unstable, sometime I got good result but most of the time the training stucked at bad local optima.

My solution: 
- Decrease learning rate
- Remove all dropout layer + reduce number of filters in all Conv2D by half.
